packages/fx-core/src/common/telemetry.ts

```diff
@@ -29,6 +29,10 @@ export enum TelemetryProperty {
   IsSideloadingAllowed = "is-sideloading-allowed",
   NeedMigrateAadManifest = "need-migrate-aad-manifest",
   CheckSideloadingHttpStatus = "check-sideloading-http-status",
+  TemplateGroup = "template-group",
+  TemplateLanguage = "template-language",
+  TemplateScenario = "template-scenario",
+  TemplateFallback = "template-fallback",
```

Comment:
In which case, we need template-fallback event?

Reply:
When scaffolding falls back to use local templates packed inside VS Code Extension.